routes/projectRoutes.js

```diff
@@ -3,6 +3,8 @@ const projectController = require('../controllers/projectController');
 
 const router = express.Router();
 
+router.post('/:slug/follow/:username', projectController.followProject);
+router.post('/:slug/unfollow/:username', projectController.unfollowProject);
```

Comment:
for unfollow, should it be router.delete instead of post?

Reply:
Now changed.